.github/workflows/validator-release.yml

```diff
@@ -0,0 +1,281 @@
+# Copyright 2022-2024, axodotdev
```

Comment:
This file is pretty dense and difficult to review with certainty. Was it based off some existing release workflow?

Reply:
It is generated by `cargo-dist`.